### PR TITLE
Initial github issue and PR templates

### PR DESCRIPTION
**Pull request by koppor**
[GitHub has a new feature "Issue and Pull Request templates](https://github.com/blog/2111-issue-and-pull-request-templates) and we directly wanted to make use of it. Here is the first shot.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+If you think, you found an issue, please provide as much information as you can.
+
+* If applicable, excerpt of the bibliography file (if applicable)
+* If applicable, screenshot, where the problematic area is marked. Check available software at [softwarecs](http://softwarerecs.stackexchange.com/q/163/18184) or directly use [GrabNDrop](http://www.scyv.de/gnd/).
+* Java version:
+* Operation system version:
```
Comment (oscargus) [inline .github/ISSUE_TEMPLATE.md]: Operating system

**Comment by oscargus** [comment]
Something about how to formulate feature requests?

**Comment by koppor** [comment]
I think, the current feature requests are formulated quite well. I don't want to let users search for hours. OK, I add a pointer to https://github.com/JabRef/jabref/wiki/FeatureRequests-Sorted, https://github.com/JabRef/jabref/labels/feature, and https://github.com/JabRef/jabref/labels/enhancement.

**Comment by koppor** [comment]
On the one hand, I am unsure whether such a strong text keeps users from submitting improvement requests. On the other hand, we have so many open issues so that a reduction of new issues is really helpful.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+### Issue Reporting
+
+If you think, you found an issue, please first check [our latest development build](http://builds.jabref.org/master/).
+If the issue still persists, please provide as much information as you can.
+
+* If applicable, excerpt of the bibliography file (if applicable)
+* If applicable, screenshot, where the problematic area is marked. Check available software at [softwarecs](http://softwarerecs.stackexchange.com/q/163/18184) or directly use [GrabNDrop](http://www.scyv.de/gnd/).
+* Java version: E.g., java version "1.8.0_66"
+* Operation system and version: Windows 8.1
```
Comment (simonharrer) [inline .github/ISSUE_TEMPLATE.md]: - JabRef version
- steps to reproduce

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+For text inspirations, consider [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request).
+
+### Checklist for a pull request
+
+Please check that you did following tasks.
+Remove them after completion, then submit the pull request.
+
+- [ ] ensure that your source branch is *not* master, but something else
+- [ ] ensure that you added your change to CHANGELOG.md
+- [ ] ensure that you followed [the formal requirements on a pull request](https://github.com/JabRef/jabref/blob/master/CONTRIBUTING.md#formal-requirements-for-a-pull-request)
```
Comment (simonharrer) [inline .github/PULL_REQUEST_TEMPLATE.md]: - tests are green
- changes explained briefly in pull request message

**Comment by koppor** [comment]
Decision in dev call: keep templates small.

**Pull request merged**